system/logaccess/ElasticStack/ElasticStackLogAccess.cpp

```diff
@@ -520,11 +603,36 @@ cpr::Response ElasticStackLogAccess::performESQuery(const LogAccessConditions &
 bool ElasticStackLogAccess::fetchLog(const LogAccessConditions & options, StringBuffer & returnbuf, LogAccessLogFormat format)
 {
     cpr::Response esresp = performESQuery(options);
-    processESJsonResp(esresp, returnbuf, format);
+    processESSearchJsonResp(esresp, returnbuf, format);
 
     return true;
 }
 
+bool ElasticStackLogStream::readLogEntries(StringBuffer & record)
+{
+    Json::Value res;
+    Json::StreamWriterBuilder builder;
```

Comment:
It is expensive to recreate this each time.  This should be a member, and builder.newStreamWriter() should be called once, and stored as a member, to avoid the overhead of creating that.